packages/gafl-webapp-service/src/handlers/attribution-handler.js

```diff
@@ -9,5 +10,12 @@ import { initialiseAnalyticsSessionData } from '../processors/analytics.js'
  */
 export default async (request, h) => {
   await initialiseAnalyticsSessionData(request)
+
+  if (request.query[UTM.CAMPAIGN] === 'renewals') {
+    if (request.query[QUERYSTRING_LICENCE_KEY]) {
+      return h.redirect(`${RENEWAL_BASE}/${request.query[QUERYSTRING_LICENCE_KEY]}`)
+    }
+    return h.redirect(`${RENEWAL_BASE}`)
```

Comment:
Remembering the work we did the other day, I think this might be better going to `IDENTIFY` RATHER THAN `RENEWAL_BASE`. The test would need to be updated as well.